src/frontend/hooks/server/deviceInfo.ts

```diff
@@ -1,10 +1,14 @@
-import {useMutation, useQuery, useQueryClient} from '@tanstack/react-query';
+import {
+  useMutation,
+  useSuspenseQuery,
+  useQueryClient,
+} from '@tanstack/react-query';
 import {useApi} from '../../contexts/ApiContext';
 
 export const useDeviceInfo = () => {
   const mapeoApi = useApi();
 
-  return useQuery({
+  return useSuspenseQuery({
```

Comment:
i don't think it makes sense to turned this shared query into a suspense query. that's why i suggested a one-off query that's only used specifically for the app navigator component. in most cases using a normal query is probably preferred, there's just a specific requirement in the context of that component

Reply:
https://github.com/digidem/comapeo-mobile/pull/313/commits/26dd5aff1325bd14753363d05ba717376de4b182

Reply:
https://github.com/digidem/comapeo-mobile/pull/313/commits/26dd5aff1325bd14753363d05ba717376de4b182